src/crkeng/collected-static/README.md

```diff
@@ -0,0 +1,8 @@
+Files in this directory are not checked in because they are collected here by
```

Comment:
I feel uneasy about `src/` containing generated/compiled output. Maybe we can tackle this during pairing!

Reply:
I will address this somewhat in a future PR.